src/Pages/Courses/Courses.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import styles from "./Courses.module.scss";
+// import book from "/images/book.png";
+// import bookmark from "/images/bookmark.png";
+// import bookmarkA from "/images/bmActive.png";
+// import backArrow from "/images/arrow.png";
+
+const Courses = () => {
+  const [coursesData, setCoursesData] = useState([]);
+  const [BM, setBM] = useState(false);
```

Comment:
using only one state to toggle bookmark feature of all courses?

If you bookmark one course, only that course should get bookmarked not all the courses.

Fix this.

Reply:
Fixed.